numpy/core/setup.py

```diff
@@ -610,7 +610,7 @@ def generate_api(ext, build_dir):
     config.add_include_dirs(join(local_dir, "src"))
     config.add_include_dirs(join(local_dir))
 
-    config.add_data_files('include/numpy/*.h')
+    config.add_data_dir('include/numpy')
```

Comment:
This will recursively add all the contents of the directory, which currently only holds `*.h` files (including the new `random/bitgen.h`